Lib/venv/__init__.py

```diff
@@ -355,11 +354,10 @@ def setup_python(self, context):
                 if os.path.lexists(src):
                     copier(src, os.path.join(binpath, suffix))
 
-            if _POSIX_BUILD:
```

Comment:
why remove this check, those extra executable are mingw specific, no?

Reply:
I thought that this change was unnecessary, but I can revert.

Reply:
yeas, please revert that part